pkg/sidecar/sidecarpb/gw_sidecar.proto

```diff
@@ -44,6 +44,11 @@ enum ClassType {
     TBF = 1;
 }
 
+enum TunnelStatusType {
+    GW_TUNNEL_STATE_UP = 0;
+    GW_TUNNEL_STATE_DOWN = 1;
+	GW_TUNNEL_STATE_UNKNOWN = 2;
```

Comment:
when would be using GW_TUNNEL_STATE_UNKNOWN ?

Reply:
Not using it, will remove